zero/server.py

```diff
@@ -83,13 +76,16 @@ def register_rpc(self, func: typing.Callable):
         if func.__name__ == "get_rpc_contract":
             raise Exception("get_rpc_contract is a reserved function; cannot have `get_rpc_contract` as a RPC function")
 
-        verify_function_args(func)
-        verify_function_input_type(func)
-        verify_function_return(func)
+
+        signature = verify_function_typing(func)
```

Comment:
`verify_function_typing` name is not clear, it verifies and returns something, doing 2 things but the name suggests it only verifies